README.md

```diff
@@ -365,3 +365,7 @@ private static void beforeUpdate_test() {
 ```
 
 Notice how we performed _zero_ DML operations yet we were able to cover all of the logic of our class in this particular test. This can help save a lot of computational time and allow for much faster execution of Apex tests.
+
```

Comment:
This is not really necessary for such a minor change. Let's omit the README updates.

Reply:
Sure @mitchspano :)

Reply:
Done @mitchspano